elpis/transformer/elan.py

```diff
@@ -178,191 +183,23 @@ def import_eaf_file(eaf_paths, context, add_annotation, tmp_dir):
             end = annotation[1]
             annotation = annotation[2]
 
-            # print("processing annotation: " + annotation, start, end)
-            obj = {
+            utterance = {
                 "audio_file_name": f"{file_name}.wav",
                 "transcript": annotation,
                 "start_ms": start,
                 "stop_ms": end
             }
-            # TODO: prehaps re-enable later
+            # TODO: re-enable later
             # if "PARTICIPANT" in parameters:
             #     obj["speaker_id"] = speaker_id
-            utterance = clean_json(obj)
-            add_annotation(file_name, utterance)
-
-
-def get_english_words() -> Set[str]:
-    """
-    Gets a list of English words from the nltk corpora (~235k words).
-    N.B: will download the word list if not already available (~740kB), requires internet.
-    :return: a set containing the English words
-    """
-    nltk.download("words")  # Will only download if not locally available.
-    return set(words.words())
 
+            utterance_cleaned = clean_json_utterance(utterance=utterance,
+                                                     remove_english=False,
+                                                     use_langid=False,
+                                                     punctuation_to_collapse_by=punctuation_to_collapse_by,
+                                                     punctuation_to_explode_by=punctuation_to_explode_by)
+            add_annotation(file_name, utterance_cleaned)
 
-def clean_utterance(utterance: Dict[str, str],
```

Comment:
I have to wonder why this kaldi_helpers code that was already in the repository was copy pasted here in the first place 😂.